packages/flame/lib/src/assets/images.dart

```diff
@@ -10,12 +10,18 @@ import 'package:flutter/services.dart';
 import '../flame.dart';
 
 class Images {
-  Images({this.prefix = 'assets/images/'})
-      : assert(prefix.isEmpty || prefix.endsWith('/'));
+  Images({String prefix = 'assets/images/'})
+      : assert(prefix.endsWith('/')),
```

Comment:
Note: empty prefixes should be allowed, that way we can let users use the full path if they want.

This is useful when using `flutter_gen` for example, which generates type safe paths which always include the whole path.

Reply:
> empty prefixes should be allowed

you say that, yet you are removing the check that was allowing empty prefixes...

Reply:
True! I totally got this the wrong way around, good catch